core/src/main/kotlin/Kotlin/KotlinLanguageService.kt

```diff
@@ -166,8 +166,17 @@ class KotlinLanguageService : CommonLanguageService() {
             keyword("dynamic")
             return
         }
+
+        val nullabilityModifier = node.details(NodeKind.NullabilityModifier).singleOrNull()
+
         if (node.isFunctionalType()) {
-            renderFunctionalType(node, renderMode)
+            nullabilityModifier ?.apply {
```

Comment:
`?.let` is better, as a receiver not changed

Reply:
agree but, in case of `?.let` if `symbol(it.name)` returns null for some reason, then `renderFunctionalType(node, renderMode)` will get executed. I will use `?.also` in this case.